OpenBeerFiller.ino

```diff
@@ -109,7 +109,7 @@ void triggerFullFillSensor1() {
  * Fired when fill sensor 1 is triggered as full.
  */
 void triggerFullFillSensor2() {
-  if (!fillSensor2Triggered && hasProgramState(FILLING)) {
+  if (!fillSensor2Triggered && hasProgramState(FILLING) && NUMBER_OF_FILLERS > 1) {
```

Comment:
If we use an array macro for the pins we can eliminate all these specific function calls into a dynamic function that just takes a pin and makes changes based on that. Same would go for triggerFullFillSensor1 and 3.